src/resolver.ts

```diff
@@ -23,3 +37,57 @@ export const resolvers = {
     },
   },
 };
+
+function thereIsNumber(str: string) {
+  return isNaN(parseFloat(str));
+}
+
+function thereIsLetter(str: string) {
+  const alf = [
+    'a',
+    'b',
+    'c',
+    'd',
+    'e',
+    'f',
+    'g',
+    'h',
+    'i',
+    'j',
+    'k',
+    'l',
+    'm',
+    'n',
+    'o',
+    'p',
+    'q',
+    'r',
+    's',
+    't',
+    'u',
+    'v',
+    'w',
+    'x',
+    'y',
+    'z',
+  ];
+  let def = false;
+  alf.forEach(function (letra) {
+    if (str.includes(letra)) {
+      def = true;
+    }
+  });
+  return def;
+}
+
+async function emailAlreadyUsed(str: string) {
+  const userRepository = AppDataSource.getRepository(User);
+  const user = await userRepository.findOneBy({
+    email: str,
+  });
+  let def = false;
+  if (user != null) {
+    def = true;
+  }
+  return def;
```

Comment:
nesses cenários você já poderia retornar direto a validação, e lembrar de utilizar !== e === ao invés de != e == (na versão "simples" o js faz umas conversões que podem levar a resultados estranhos)
```suggestion
  return user !== null;
```